GOVERNANCE.md

```diff
@@ -0,0 +1,94 @@
+
+Subsequently, the library received significant contributions from LAAS-CNRS (Arthur Bit-Monnot), the University of Brescia (Alberto Rovetta) and the University of Rome "La Sapienza" (Alessandro Trapasso).
+
+Up to June 30th 2023, the development of the library has been coordinated by Andrea Micheli (in the role of work-package leader in the AIPlan4EU project), discussing with the AIPlan4EU project partners the technical and design choices in a bi-weekly remote meeting.
+
+Starting from November 1st 2023, the UP project is ruled according to the prescriptions in this document.
+
+
+# General Governance Spirit
+
+The UP project wants to implement a ["do-ocracy"](https://www.redhat.com/en/blog/understanding-open-source-governance-models): people actively contributing to the development should be the ones allowed to make the decisions.
+
+This general principle will apply from the technical level (e.g. by determining who can accept or reject external contributions and have the commit permissions) to the strategic level (e.g. by determining who can decide if a certain feature needs to be developed or is out of scope).
+
+This document describes a lightweight set of procedures designed to implement this general spirit as effortlessly as possible for the stakeholder involved.
+
+
+# Roles
+
+We only recognize two roles in the UP project:
+
+- **Committer**: a person who can create new branches and commit on any branch except the "master" branch;
+- **Maintainer**: a Committer who can also commit on the "master" branch. A list of all active Maintainers is always kept up-to-date in the [MAINTAINERS](/MAINTAINERS) file.
+
+In addition, being an open source project we welcome external contributions coming from external forks of the UP project. We will refer to such Pull-Requests as "External Contributions".
+
+The **Board of Maintainers** is the main decision body of the UP project and is composed of all the current Maintainers.
+
+
+# Communication Channels
+
+We establish two official communication channels for the UP project in addition to the GitHub facilities (issues, discussions, PRs).
+
+- The **Maintainers Mailing List** (unified-planning-maintainers@googlegroups.com): all and only the active Maintainers are part of this list. The mailing list is not publicly readable but anyone can send messages to it. It is the main and official communication channel among maintainers and to communicate to the board of maintainers.
+
+- The **Public Mailing List** (unified-planning@googlegroups.com): is a general-purpose communication channel to be used for both technical matters and public announcements. The list is publicly readable and anyone can join it. The outcome of all the votes of the Board of Maintainers (required for the situations indicated in the following) are publicly communicated through this list.
```

Comment:
And also anyone can send message to it? If anyone can send messages, as I suppose it is, are messages supposed to be accepted/rejected by someone when received before getting public?

Reply:
Good point. I was thinking on a list where anyone can subscribe and post without moderation (except spam), but then we need a ban mechanism for trolls... We can discuss this at the enxt meeting